src/lib/index.ts

```diff
@@ -101,6 +102,8 @@ const getDelta = async(snykTestOutput: string = '', debugMode = false):Promise<S
       
       newVulns = typedSnykTestJsonResults.vulnerabilities.filter(x => x.type != "license")
       newLicenseIssues = typedSnykTestJsonResults.vulnerabilities.filter(x => x.type == "license")
+
+      passIfNoBaseline = true
```

Comment:
That should be an option set by the user. Some folks will want to have the default behavior while some others will want to pass if no baseline